server/users.go

```diff
@@ -45,20 +57,39 @@ type userResponse struct {
 	Name     string    `json:"name"`
 	Provider string    `json:"provider"`
 	Scheme   string    `json:"scheme"`
+	Roles    []string  `json:"roles"`
 }
 
 func newUserResponse(u *chronograf.User) *userResponse {
+	roles := make([]string, len(u.Roles))
+	for i, r := range u.Roles {
+		roles[i] = r.Name
+	}
 	return &userResponse{
 		ID:       u.ID,
 		Name:     u.Name,
 		Provider: u.Provider,
 		Scheme:   u.Scheme,
+		Roles:    roles,
 		Links: selfLinks{
 			Self: fmt.Sprintf("/chronograf/v1/users/%d", u.ID),
 		},
 	}
 }
 
+// ExplicatedRoles fills out a set of roles to include its members explicitly
+func ExplicatedRoles(reqRoles []string) ([]chronograf.Role, error) {
```

Comment:
No need for this if you use []chronograf.Role

Reply:
👍 thanks - vestigial as per above